neos/neos-ui
# FEATURE: set Into as the default insertion mode for the content tree

Resolves: https://github.com/neos/neos-ui/issues/2776

**What I did**

Set Into as the default insertion mode for the content tree.

**How to verify it**

Try to insert content and document nodes. Into should be the default position for content nodes, if it's allowed.
